src/main/java/cz/muni/fi/pa165/entity/User.java

```diff
@@ -0,0 +1,22 @@
+package cz.muni.fi.pa165.entity;
+
+/**
+ * Hello world!
+ *
+ */
+
+@Entity
+@Table(name="User")
```

Comment:
Není to zbytečný, resp. nebude třeba tu tabulku stejně přejmenovat na Users (kvůli derby, testům a tak)?